src/renderer/widgets/DelegateModal/lib/helpers.ts

```diff
@@ -12,3 +13,7 @@ export const getTreasuryTrackDescription = (asset: Asset | null, description: st
 
   return t(`${description}General`);
 };
+
+export const getTrackIds = (tracks: Track[], votedTracks: string[]): number[] => {
+  return tracks.filter((t) => !votedTracks.includes(t.id)).map((track) => Number(track.id));
+};
```

Comment:
This check for voting is already covered by this sample in the model. Don't need for another filtering function here

```
sample({
  clock: tracksSelected,
  source: { tracks: $tracks, votedTracks: $votedTracks },
  fn: ({ tracks, votedTracks }, newTracks) => {
    const resultArray = newTracks.filter((num) => !votedTracks.includes(num.toString()));

    if (resultArray.every((t) => tracks.includes(t))) {
      return removeItemsFromCollection(tracks, resultArray);
    }

    return addUniqueItems(tracks, resultArray);
  },
  target: $tracks,
});
```

Reply:
Only this checking isn't enough, because we should show chips as selected when select all available tracks